From c845d8639e5902c310a1ca064cc4438163ae6b55 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sun, 29 Dec 2019 17:28:08 +0200
Subject: [PATCH 05/13] replace Gallery2 app

	modified:   target/product/handheld_product.mk
	modified:   target/product/mainline_arm64.mk
---
 target/product/handheld_product.mk | 2 +-
 target/product/mainline_arm64.mk   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/target/product/handheld_product.mk b/target/product/handheld_product.mk
index 08b592e..57734c4 100644
--- a/target/product/handheld_product.mk
+++ b/target/product/handheld_product.mk
@@ -28,7 +28,6 @@ PRODUCT_PACKAGES += \
     Contacts \
     DeskClock \
     Email \
-    Gallery2 \
     Launcher3QuickStep \
     Music \
     OneTimeInitializer \
@@ -36,6 +35,7 @@ PRODUCT_PACKAGES += \
     QuickSearchBox \
     Settings \
     SettingsIntelligence \
+    SimpleGallery \
     StorageManager \
     SystemUI \
     WallpaperCropper \
diff --git a/target/product/mainline_arm64.mk b/target/product/mainline_arm64.mk
index f21fa89..1f8aad0 100644
--- a/target/product/mainline_arm64.mk
+++ b/target/product/mainline_arm64.mk
@@ -38,11 +38,11 @@ PRODUCT_ARTIFACT_PATH_REQUIREMENT_WHITELIST += \
   system/app/DeskClock/oat/arm64/DeskClock.odex \
   system/app/DeskClock/oat/arm64/DeskClock.vdex \
   system/app/Email/Email.apk \
-  system/app/Gallery2/Gallery2.apk \
   system/app/LatinIME/LatinIME.apk \
   system/app/LatinIME/oat/arm64/LatinIME.odex \
   system/app/LatinIME/oat/arm64/LatinIME.vdex \
   system/app/Music/Music.apk \
+  system/app/SimpleGallery/SimpleGallery.apk \
   system/app/QuickSearchBox/QuickSearchBox.apk \
   system/app/webview/webview.apk \
   system/bin/healthd \
-- 
2.17.1

